<?xml version="1.0" encoding="UTF-8"?><testrun name="Q5Test" project="Question5" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="com.revature.q5.test.Q5Test" time="2.235">
    <testcase name="testReducer" classname="com.revature.q5.test.Q5Test" time="0.646"/>
    <testcase name="testMapper" classname="com.revature.q5.test.Q5Test" time="1.584">
      <failure>java.lang.AssertionError: 9 Error(s): (Missing expected output (United Kingdom Agriculture Female, 1.61000001430511) at position 4., Missing expected output (United Kingdom Agriculture Female, 1.5900000333786) at position 2., Missing expected output (United Kingdom Agriculture Female, 1.71000003814697) at position 0., Missing expected output (United Kingdom Agriculture Female, 1.52999997138977) at position 3., Missing expected output (United Kingdom Agriculture Female, 1.64999997615814) at position 1., Received unexpected output (United Kingdom Agriculture Female, -3.508775476627975) at position 0., Received unexpected output (United Kingdom Agriculture Female, 5.2287610726340255) at position 3., Received unexpected output (United Kingdom Agriculture Female, -3.7735887251106175) at position 2., Received unexpected output (United Kingdom Agriculture Female, -3.636360221000972) at position 1.)
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.mrunit.TestDriver.validate(TestDriver.java:294)
	at org.apache.hadoop.mrunit.MapDriverBase.runTest(MapDriverBase.java:151)
	at org.apache.hadoop.mrunit.TestDriver.runTest(TestDriver.java:137)
	at com.revature.q5.test.Q5Test.testMapper(Q5Test.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
    <testcase name="testMapReduce" classname="com.revature.q5.test.Q5Test" time="0.004"/>
  </testsuite>
</testrun>
